Classes/source.rb

```diff
@@ -0,0 +1,21 @@
+class Source
+  attr_accessor :id
+  attr_reader :book_source, :items
+
+  def initialize(book_source)
+    @book_source = book_source
+    @items = []
+    @id = Random.rand(1..1000)
+  end
+
+  def add_item(item)
+    @items << item
+    item.book_source = self
+  end
+
+  def as_hash
+    {
+      'book_source' => @book_source
+    }
+  end
+end
```

Comment:
- [ ] As per [this given requirement](https://github.com/microverseinc/curriculum-ruby/blob/main/group-capstone/catalog_of_my_things_divided.md#:~:text=Team%20member%20%234%20(if%20your%20team%20has%203%20members%20%2D%20skip%20those%20requirements)), you're to skip these requirements if the team has less than 4 members. Kindly delete this class.

Reply:
@ShadyShawkat,
thanks for your good and valuable feedback, as you mention the source class removed.